test/boulder-tools/build.sh

```diff
@@ -22,7 +22,11 @@ apt-get install -y --no-install-recommends \
   ruby-dev \
   rsyslog \
   softhsm \
-  protobuf-compiler &
+  protobuf-compiler \
+  build-essential \
```

Comment:
`build-essential` brings in a lot of large packages. You should uninstall it, and the packages it implicitly pulls in, once the script is done.